src/client/options/mod.rs

```diff
@@ -528,6 +549,7 @@ impl From<ClientOptionsParser> for ClientOptions {
             socket_timeout: parser.socket_timeout,
             zlib_compression: parser.zlib_compression,
             direct_connection: parser.direct_connection,
+            driver_info: None,
```

Comment:
Given that the purpose of this is for libraries wrapping the driver to add info about themselves, I don't think it's super likely that anyone will want to specify these options from the URI, so I didn't implement any URI options corresponding to `DriverInfo` fields.

Reply:
should we mention this in the docstrings for the options?

Reply:
Mention that this should only be specified by wrapping libraries, or that there's no URI options mapping to it?

Reply:
I meant the latter, and the docs in the PR already mention the former. Since [we direct users](https://github.com/mongodb/mongo-rust-driver/blob/master/src/client/mod.rs#L84) specifically to `ClientOptions::parse` (which does not mention this option), I don't actually think there's anything to do here though.